action.yml

```diff
@@ -1,35 +1,28 @@
 name: "Sencha App Builder"
 description: "Build the frontend distribution for MAGIQ Frontend apps"
 inputs:
-  sencha-version:
+  sdk-prefix-with-file:
     description: "Sencha SDK Version"
     required: true
-  bucket-with-prefixes:
-    description: "location where the artifacts will be pushed to"
-    required: true
-  sencha-sdk-bucket:
+  sdk-bucket:
     description: "where the SDKS are located"
     required: true
   build-files:
     description: "required build files"
     required: true
+  artifact-bucket:
+    description: "location where the artifacts will be pushed to"
+    required: true
+  artifact-prefix:
+    description: "The prefix location of the bucket"
+    required: true
   build-folder:
     description: "what folder do you want the output to be in"
     default: build
+  environment-build:
+    description: is a production build
+    default: "dev"
 runs:
-  using: "composite"
-  steps:
-    - run: echo ${{inputs.sencha-version}}
-      shell: bash
-    - run: aws s3 cp s3://${{inputs.build-files}} . --recursive
-      shell: bash
-    - run: aws s3 cp s3://${{inputs.sencha-sdk-bucket}}/extjs-${{inputs.sencha-version}}.tgz extjs-${{inputs.sencha-version}}.tgz
-      shell: bash
-    - run: ls -al
-      shell: bash
-    - run: docker build -t sencha-app --build-arg senchaVersion=${{inputs.sencha-version}} --output ./${{inputs.build-folder}} .
-      shell: bash
-    - run: tar -zcvf ${{inputs.build-folder}}.tar.gz ${{inputs.build-folder}}
-      shell: bash
-    - run: aws s3 cp ${{inputs.build-folder}}.tar.gz s3://${{inputs.bucket-with-prefixes}}/2.3.${{ github.run_number }}
-      shell: bash
+  using: 'node12'
```

Comment:
Can we use Node.js 16 Active Long Term Support (LTS)?

Reply:
done.